### DISCUSSION TITLE
🐞 Cannot use `dagger -s call ... terminal`

### DISCUSSION THREAD

**Discussion by jpadams** [dagger/dagger#12198] [Help]
### What is the issue?

I was running some functions repeatedly with `dagger -s` to get less verbose output to my terminal. I wanted concise output to share with colleagues as a repro. My `dagger -s call` commands were working fine until I added `terminal` to the end of a call that returned a `Container` type. I expected it to work as normal, but dagger threw an error.

An example:
```
Error: response from query: input: container.from.terminal resolve: failed to forward exit code: failed to close terminal: EOF
```

### Dagger version

dagger v0.13.3

### Steps to reproduce

Repros
```
dagger-cache-volume-issue ➤ dagger -s call populate-cache --input "foo" terminal
Error: response from query: input: container.from.withMountedCache.withExec.withExec.withExec.withExec.withExec.withExec.withExec.withExec.terminal resolve: failed to forward exit code: failed to close terminal: EOF

Run 'dagger call populate-cache terminal --help' for usage.
```
dagger -s core container from --address alpine terminal
```
vs
```
dagger core container from --address alpine terminal
```



### Log output

see above

**Comment by jedevc** [comment]
This does appear to be intentional: https://github.com/dagger/dagger/blob/701be1ccab448437e0f03b1c1461c4b1456a0051/cmd/dagger/terminal.go#L18-L20

There should be a better error message presented here (instead of nothing), but I don't think it is intended to work.

**Comment by bevel-zgates** [comment]
Just ran into this myself. This would be really helpful until #10401 gets implemented for a few of my workflows that take a lot of different input params. Specifically the passing of secrets that I cannot do without passing each as a param.

As a workaround you can just grab the latest layer and run it directly `docker run -it --rm <layer> /bin/bash`
